test/disability-benefits/526EZ/config/privateRecordReleases.unit.spec.jsx

```diff
@@ -38,23 +42,27 @@ describe('Disability benefits 526EZ VA facility', () => {
         uiSchema={uiSchema}/>
     );
 
-    fillData(form, 'select#root_privateRecordReleases_0_privateRecordRelease_startTreatmentMonth', '1');
-    fillData(form, 'select#root_privateRecordReleases_0_privateRecordRelease_startTreatmentDay', '3');
-    fillData(form, 'input#root_privateRecordReleases_0_privateRecordRelease_startTreatmentYear', '1950');
-    fillData(form, 'select#root_privateRecordReleases_0_privateRecordRelease_endTreatmentMonth', '1');
-    fillData(form, 'select#root_privateRecordReleases_0_privateRecordRelease_endTreatmentDay', '3');
-    fillData(form, 'input#root_privateRecordReleases_0_privateRecordRelease_endTreatmentYear', '1955');
-    fillData(form, 'input#root_privateRecordReleases_0_privateRecordRelease_treatmentCenterName', 'Local facility');
+    fillData(form, 'select#root_privateRecordReleases_0_treatmentDateRange_fromMonth', '1');
```

Comment:
Now that we are using the standard date widget, we could use the fillDate helper here, if desirable.

Reply:
Oh, see, I didn't even know that existed! Seems like a good idea.